evap/results/tests/test_views.py

```diff
@@ -52,6 +52,22 @@ def test_multiple_evaluations_per_course(self):
         self.assertNotContains(page, evaluation2.full_name)
         caches['results'].clear()
 
+
```

Comment:
We try to style our code according to [pep8](https://www.python.org/dev/peps/pep-0008/#blank-lines), which says that
> Method definitions inside a class are surrounded by a single blank line.

So it would be cool if you could remove one blank line before and after this test